components/automate-cli/pkg/verifyserver/services/systemresourceservice/getosandfilesysteminfoutil_test.go

```diff
@@ -0,0 +1,29 @@
+package systemresourceservice
+
+import (
+	"runtime"
+	"testing"
+
+	"github.com/bmizerany/assert"
```

Comment:
Use `github.com/stretchr/testify/assert` instead of `github.com/bmizerany/assert`.

Reply:
Done